src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -1556,6 +1556,24 @@ internal CommonAssemblyWellKnownAttributeData GetSourceDecodedWellKnownAttribute
             return (CommonAssemblyWellKnownAttributeData)attributesBag.DecodedWellKnownAttributeData;
         }
 
+        // This only forces binding of attributes that look like they may be forwarded types attributes (syntactically).
+        internal HashSet<NamedTypeSymbol> GetForwardedTypes()
+        {
+            CustomAttributesBag<CSharpAttributeData> attributeBag = null;
+            LoadAndValidateAttributes(OneOrMany.Create(GetAttributeDeclarations()), ref attributeBag, attributeMatchesOpt: this.IsForwardedTypesAttribute);
```

Comment:
>LoadAndValidateAttributes(OneOrMany.Create(GetAttributeDeclarations()), ref attributeBag, attributeMatchesOpt: this.IsForwardedTypesAttribute); [](start = 12, length = 143)

Is it possible that this is going to cause duplication of errors for the attributes that we try to bind?

Reply:
It was indeed. Thanks!

---
In reply to: [155961411](https://github.com/dotnet/roslyn/pull/23647#discussion_r155961411) [](ancestors = 155961411)